solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -1146,6 +1146,11 @@
       <str>yyyy-MM-dd HH:mm[:ss[.SSS]][z</str>
       <str>yyyy-MM-dd HH:mm[:ss[,SSS]][z</str>
       <str>yyyy-MM-dd</str>
+      <str>EEE, dd MMM yyyy HH:mm:ss z</str>
+      <str>EEEE, dd-MMM-yy HH:mm:ss z</str>
+      <str>EEE MMM d HH:mm:ss yyyy</str>
+      <str>EEE MMM d hh:mm:ss z yyyy</str>
```

Comment:
The "hh" is erroneous.  I'm surprised this even works; perhaps "lenient" allows os to get away with it?  I was hoping to keep the added patterns more minimal.  You recently added "parse-date-patterns-from-extract-contrib" to solrconfig-parsing-update-processor-chains which is a better guide.

Reply:
Should I just try and copy that part of the config to the default one?

Reply:
Yeah, that's what I thought you were going to do.  You've separately tested that the ExtractionDateUtils tests also work on these patterns in the last issue, so we want them to be the same; otherwise due to bugs (like your 'hh') we might inadvertently screw that up.  If needed, modify/tweak one of those tests/configs.